### ISSUE TITLE
Quick UAT check on March 23 on FMI

### ISSUE THREAD

**Issue by Ye-walmart** [Ye-walmart/Leapfrog_FMI#17]
I would like to have a quick UAT again in the afternoon of March 23, so before doing this test.  Can we align on the what's the expectation of the product feature?  Here is my list of expectation. 
@vera3224 @fuzhuang @BeanlyViwor Please feedback on my expectation list and could add or specify it if needed. 

Item content: 

- [x] - 90%-95% coverage rate
- [x] - item name: brandname +productname
- [ ] - no GHS special information on the product name

category

- [x] - no duplicated categories
- [x] - no promotion categories
- [x] - no wrong items in categories

search

- [x] - improved relevance
- [x] - hot search recommendation is hidden 

map

- [x] - fresh items with no mod category will only show section map 
- [x] - non fresh items with mod category will be able to zoom in and show category map
- [x] - highlight effect and some special icons
- [x] - bubble will not hide the category information

bookmark

- [x] - expected action feedback
- [x] - able to only delete items from one label not from all labels
- [x] - no obvious latency and crunches
- [x] - "早餐的使命' banner. Do you have a good quality banner or not please hide it

store locator

- [x] - able to select cites and stores

others:

- [x] - mini-program icon: new 沃尔玛找找货 icon
- [x] - mini-program name: 沃尔玛找找货

**Comment by fuzhuang** [comment]
@Ye-walmart  Some detailed bookmark UX has not been confirmed before, what we implemented now is to delete item from all tags once it's deleted. Let's review the UX after MVP and prioritize needed changes.

**Comment by BeanlyViwor** [comment]
item (which from DAS ) Name have been fixed to brandName+productName;

**Comment by BeanlyViwor** [comment]
Promotion category have been closed by Viwor.

**Comment by BeanlyViwor** [comment]
@fuzhuang Could you please kindly export the item list (UPC) frontend, so we can check the item coverage rate.

**Comment by BeanlyViwor** [comment]
BTW, about the GHS special information on the product name, @makara  please check is there any problem about fetching data from DAS, we update some items and the system has sent NOTIFY automatically, however the data front-end is still the old version.

**Comment by makara** [comment]
Hi @BeanlyViwor, I can't tell exactly. We've been doing a lot transferring between dev and staging, the code, the data, the settings, etc etc. It might be lost in between. Instead of us trying to restore, it might be safer to have you push it again, or even safer, have a full initialize strategy and let us initialize from scratch - we can discuss.

**Comment by BeanlyViwor** [comment]
@makara I agree. Let's have a quick talk by wechat later, when you are available please at me ASAP.

**Comment by BeanlyViwor** [comment]
@Ye-walmart Could you please kindly check is there any items in the wrong category？we will track and find out is this ticket could be colsed.  @fuzhuang

**Comment by Ye-walmart** [comment]
@BeanlyViwor The previous cases of wrong categories have been solved. I haven't seen any more cases so far.

**Comment by BeanlyViwor** [comment]
duplicated categories problem has been resolved this morning.
